jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -288,4 +298,8 @@ private Variant loadVariantWhenAuthorized(
         // unauthorized...
         return null;
     }
+
+    private String extractPushIdentifier(HttpServletRequest request) {
+        return request.getHeader("push-identifier");
```

Comment:
that would complicate the SDKs, and therefore is more error thrown. I'd prefer the server figuring this out